test/unit/test_utils.js

```diff
@@ -15,6 +15,14 @@
 
 import { CMapCompressionType } from '../../src/shared/util';
 
+class NodeFileReader {
+  constructor(params) {
+    var fs = require('fs');
+    var file = fs.readFileSync(params.path);
+    return new Uint8Array(file);
```

Comment:
I'm having a bit of trouble with the constructor returning a value. Usually constructors don't return anything, so it's a bit confusing that this apparantly works in JavaScript. Could we perhaps create a different structure for this, for example adding a method `read` or something like that?